### ISSUE TITLE
How to make the body parameters hidden on the html page

### ISSUE THREAD

**Issue by Nicolas-Mousten-WeCode** [knuckleswtf/scribe#751]
### Scribe version

4.23

### Your question

I have read the documentation and earlier questions where i found that no-example could maybe work but it only removed the body in the request. 

Is there any other way to make sure that the docs dosen't have the section with body parameters?

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
Nope, no way except for just not adding them. But you can definitely create a custom strategy or custom themes or something. What's your use case?

**Comment by rizvikobemotor** [comment]
The use case is here is my laravel form request, and I don't need to show the active body parameter to docs UI because I don't want to get user input

`<?php

namespace App\Http\Requests\Auth;

use Illuminate\Foundation\Http\FormRequest;

class LoginRequest extends FormRequest
{
    public function authorize(): bool
    {
        return true;
    }

    public function rules(): array
    {
        return [
            'email' => ['required', 'email:rfc,dns', 'exists:users'],
            'password' => ['required', 'min:8'],
            'active' => ['present'],
        ];
    }

    protected function prepareForValidation(): void
    {
        $this->merge([
            'active' => 1,
        ]);
    }
}
`

<img width="662" alt="Screenshot 2024-02-21 at 10 53 38 AM" src="https://github.com/knuckleswtf/scribe/assets/142575931/ad92bd01-f9be-4569-8ee3-d238196c94f7">
